src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -110,16 +110,23 @@ class IndexLogManagerImpl(indexPath: Path) extends IndexLogManager with Logging
     }
   }
 
-  override def createLatestStableLog(id: Int): Boolean = {
-    // TODO: make sure log with the id has a stable state.
-    try {
-      FileUtil.copy(fs, pathFromId(id), fs, latestStablePath, false, new Configuration)
-    } catch {
-      case ex: Exception =>
-        logError(s"Failed to create the latest stable log with id = '$id'", ex)
+  override def createLatestStableLog(id: Int): Boolean =
```

Comment:
nit: can we put `{` `}` around this `def`?

Reply:
done